jin/(7월 1주차 과제)/1037.py

```diff
@@ -0,0 +1,5 @@
+n = int(input())
+a = list(map(int, input().split()))
+a_max = max(a)
+a_min = min(a)
+print(a_max * a_min)
```

Comment:
```suggestion
print(min(a) * max(a))
```
이런 식으로 바로 결과를 출력해도 될 것 같아요